metricbeat/Makefile

```diff
@@ -22,21 +22,22 @@ include ${ES_BEATS}/libbeat/scripts/Makefile
 # Collects all module dashboards
 .PHONY: kibana
 kibana:
-	-rm -r etc/kibana
+	# To not remove index-pattern as generated by update
+	-rm -r etc/kibana/dashboard etc/kibana/search etc/kibana/visualization
 	mkdir -p etc/kibana
 	-cp -r module/*/_meta/kibana etc/
 
 # Collects all module and metricset fields
 .PHONY: fields
 fields:
 	mkdir -p etc/
-	cat ${ES_BEATS}/metricbeat/etc/_meta/fields_base.yml > etc/fields.yml
-	. ${PYTHON_ENV}/bin/activate; python ${ES_BEATS}/metricbeat/scripts/fields_collector.py >> etc/fields.yml
+	cat ${ES_BEATS}/metricbeat/etc/fields.yml > etc/fields.generated.yml
```

Comment:
@ruflin I am not sure I understand the reason for replacing `_meta/fields_base.yml` with `fields.yml`. It supposed to contain common fields that are exported by all Metricbeat modules, right? If yes, then wouldn't make more sense to call it something more obvious  like `etc/fields_common.yml`?